coldFilmSearchParser.js

```diff
@@ -0,0 +1,70 @@
+const htmlparser = require('htmlparser2');
+
```

Comment:
too long function
it's hard to read it
and will be hard to maintain
don't forget to make sub-functions with good names